practice/src/assignment1/Main.kt

```diff
@@ -0,0 +1,32 @@
+package assignment1
+
+import java.util.*
+
+fun main() {
+
+    val sc = Scanner(System.`in`)
+    val N = sc.nextInt()
+
+    for (i in 1..N) {
+        var stack = Stack<Char>()
+        val ps = sc.next()
+        var result: Boolean = true
+        for (p in ps) {
+            if (p == '(') {
+                stack.push(p)
+            } else {
+                if (stack.size() == 0) {
+                    result = false
+                    break
+                }
+                stack.pop()
+            }
+        }
+        if (result == true && stack.size() == 0) {
```

Comment:
```suggestion
        if (result && stack.size() == 0) {
```

이런식으로 boolean은 바로 조건식에서 and 연산 갈길 수 있습니다!